app/uk/gov/nationalarchives/omega/editorial/models/EditSet.scala

```diff
@@ -33,3 +37,35 @@ case class EditSetRecord(
   startDate: String,
   endDate: String
 )
+object EditSetRecord {
+  implicit val editSetRecordReads: Reads[EditSetRecord] =
+    (
+      (JsPath \ "ccr").read[String](minLength[String](5)) and
+        (JsPath \ "oci").read[String](minLength[String](5)) and
+        (JsPath \ "scopeAndContent").read[String](minLength[String](10)) and
+        (JsPath \ "coveringDates").read[String](minLength[String](0)) and
+        (JsPath \ "formerReferenceDepartment").read[String](minLength[String](0)) and
+        (JsPath \ "startDate").read[String](minLength[String](0)) and
+        (JsPath \ "endDate").read[String](minLength[String](0))
+    )(EditSetRecord.apply _)
+}
```

Comment:
You can just do `implicit val editSetRecordReads: Reads[EditSetRecord] = Json.reads[EditSetRecord]` but I see you've added some validation to these fields. We need to be a bit careful about this, for example scope and content can be empty. Also I don't think `minLength[String](0)` would have any effect. We also need to think carefully about where we do validation as if we do it on more than one place we could end up having different values acceptable in different places. Let's discuss..